src/appleseed/renderer/modeling/material/material.cpp

```diff
@@ -87,6 +87,8 @@ Material::Material(
     set_name(name);
 
     m_inputs.declare("surface_shader", InputFormatEntity, "");
+
+    m_cast_shadows = true;
```

Comment:
I would remove the option to disable cast shadows for materials. 
The option should be in the EDF class, but this can be hard to do in some cases, like OSL shaders with emission.
We can think about how can we do it and leave this part for later

Reply:
Ok. I'll make the changes.